modules/aws/lb/variables.tf

```diff
@@ -0,0 +1,321 @@
+# Load Balancer Variables
+variable "name" {
```

Comment:
Consider whether we want to use `name` or `name_prefix`. My rule of thumb has been that if there could be multiples of something I default to using `name_prefix` as they avoid collisions at the outset. Rare that we care if the name includes the datetime.

Reply:
I would like to talk more about this. I think when it comes to global name spaces it makes sense, but maybe we can make it optional here?

just initial thoughts. going to come back to this.